Xamarin.Forms.Core/RadioButton.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+    [RenderWith(typeof(_RadioButtonRenderer))]
+    public class RadioButton : View, IFontElement, ICheckedController
+    {
+        public static readonly BindableProperty TextProperty = BindableProperty.Create("Text", typeof(string), typeof(RadioButton), null,
+          propertyChanged: (bindable, oldVal, newVal) => ((RadioButton)bindable).InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged));
+
+        public static readonly BindableProperty TextColorProperty = BindableProperty.Create("TextColor", typeof(Color), typeof(RadioButton), Color.Default);
+
+        public static readonly BindableProperty FontProperty = BindableProperty.Create("Font", typeof(Font), typeof(RadioButton), default(Font), propertyChanged: FontStructPropertyChanged);
```

Comment:
In most controls, we obsoleted the Font property in favor of family, size, attribute. I would not add this.